prometheus/base/alerting/prometheus.argoCD_alerts.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    openshift.io/prometheus-rule-evaluation-scope: leaf-prometheus
+  name: o11y-argoCD-alerting-rules
+  namespace: o11y
+spec:
+  groups:
+  - name: argoCD_alerts
+    rules:
+    - alert: DegradedArgoCDApp
+      expr: max_over_time(argocd_app_info{health_status="Degraded", dest_namespace!~".+-tenant"}[5m]) == 1
```

Comment:
If the application was degraded and then got fixed the output would look like:
1 1 0 0 0.
Using max_over_time for it will return true since the max value in this time period is 1, but the application was not degraded for the past 5 minutes (only for the first 2 minutes).  
Am I right?

Reply:
as for my understanding of the task, the rule should alert if an app's status was changed to "Degraded" in the last5 minutes. However, there's a chance I misunderstood the task.
@yftacherzog What do you think? maybe that's why it was "last_over_time" instead of "max_over_time"?

Reply:
@ShakedAviv50313 when we looked at real data together, I don't think we ever saw a value of zero. It was only one or none. With that in mind, I think we need at least some of the test cases to represent that. If I understand correctly, none means the value is missing, which in promtool is represented by an underscore. 

See this link: https://prometheus.io/docs/prometheus/latest/configuration/unit_testing_rules/#series

Reply:
I think `max_over_time` makes more sense here.

Reply:
Sure, it is now done and I added a new test case representing a flappy app with also None "_" values